### ISSUE TITLE
Feature request — native Mac notifications

### ISSUE THREAD

**Issue by andrewmccullough** [oklai/koala#384]
It’s very annoying to have to switch back to the Koala window whenever there is a notification that my files compiled successfully or encountered an error. You should look into using built-in notifications on OS X for a less intrusive way to tell the user that their files have or haven’t compiled.

**Comment by oklai** [comment]
Yes, I plan to use [node-notify](https://github.com/mikaelbr/node-notifier) replace the existing methods.

**Comment by ryanbowden** [comment]
does this mean, for windows 8 and above it will use native notifications as well?

My Concern if so, is unless you can change the colour may not know if there an issue.
